static/app/views/monitors/create.tsx

```diff
@@ -11,12 +14,22 @@ import {Monitor} from './types';
 type Props = AsyncView['props'] & RouteComponentProps<{orgId: string}, {}>;
 
 export default class CreateMonitor extends AsyncView<Props, AsyncView['state']> {
+  static contextTypes = {
+    router: PropTypes.object,
```

Comment:
This router context doesn't seem to be used anywhere.

Reply:
It isn't used here, but parts of AsyncComponent use `router` so it needs to be here too.